src/api/Storage.ts

```diff
@@ -4,7 +4,8 @@ const PREVIOUS_CUR_LIBS_KEY = `prevCurLibs`;
 const LAST_PROFILE_KEY = `currentProfile`;
 const SOURCE_LIST_KEY = `sourceList`;
 const DEPLOYMENT_KEY = `deployment`;
-const DEBUG_KEY = `debug`
+const DEBUG_KEY = `debug`;
+const SERVER_SETTINGS_CACHE_KEY = `serverSettingsCache`;
```

Comment:
Since your setting's key depends on a name, make this a function and use it later on, so you don't repeat the key contruction code over and over.
```suggestion
const SERVER_SETTINGS_CACHE_KEY = (name : string) => `serverSettingsCache_${name}`;
```